tests/test_routes.py

```diff
@@ -8,11 +8,12 @@
 import os
 import logging
 from unittest import TestCase
-from unittest.mock import MagicMock, patch
+#from unittest.mock import MagicMock, patch
```

Comment:
If not needed, please remove it.
In fact, it looks like you are modifying code from the master branch. Make sure it is not used before you remove it.